packages/sourcemapped/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "sourcemapped",
```

Comment:
Should this be `@vercel/sourcemapped`?

Reply:
We prefix `@vercel/ncc` and `@vercel/nft` so it makes sense to prefix this too. Otherwise it wont be published to the vercel scope and we need to figure out permissions. Furthermore, its not really a public tool in the same way `vercel` cli or `next` are.